Identity documents endpoints (#699)
MewsSystems/gitbook-connector-api: operations/README.md

| [Update customer](customers.md#update-customer) | Updates personal information of a customer |
| [~~Merge customers~~](customers.md#merge-customers) | **Deprecated!** Please use [Merge accounts](accounts.md#merge-accounts) instead. |
| [Add customer file](customers.md#add-customer-file) | Attaches the specified file to the customer profile |

> | [Get all identity documents](../operations/identitydocuments.md#get-all-identity-documents) | Returns all identity documents for selected customers |

Reviewer: ```suggestion
| [Get all identity documents](../operations/identitydocuments.md#get-all-identity-documents) | Returns all identity documents for the specified customers |
```

> | [Add identity documents](../operations/identitydocuments.md#add-identity-documents) | Adds new identity documents to the customer |

Reviewer: ```suggestion
| [Adds identity documents](../operations/identitydocuments.md#add-identity-documents) | Adds new identity documents |
```

> | [Update identity documents](../operations/identitydocuments.md#update-identity-documents) | Updates information of the identity documents |

Reviewer: ```suggestion
| [Update identity documents](../operations/identitydocuments.md#update-identity-documents) | Updates specified identity documents |
```

> | [Delete identity documents](../operations/identitydocuments.md#delete-identity-documents) | Deletes selected identity documents |

Reviewer: ```suggestion
| [Delete identity documents](../operations/identitydocuments.md#delete-identity-documents) | Deletes specified identity documents |
```

> | [Clear identity documents](../operations/identitydocuments.md#clear-identity-documents) | Deletes all identity documents for selected customers. |

Reviewer: ```suggestion
| [Clear identity documents](../operations/identitydocuments.md#clear-identity-documents) | Deletes all identity documents for the specified customers |
```
